azurerm/resource_arm_app_service.go

```diff
@@ -29,6 +29,33 @@ func resourceArmAppService() *schema.Resource {
 				ValidateFunc: validateAppServiceName,
 			},
 
+			"identity": {
```

Comment:
Can we document these properties in the documentation? That's stored in this file, it'll want the `type` property documented in the `Argument Reference` block and the `principal_id` and `tenant_id` fields documented in the `Attributes Reference` block

Reply:
done